voxter/slate
# Dev (#3)

* Update rouge languages link in README

* Add PR template

* Add issue template

* Update readme

* Remove CONTRIBUTING

* Bump CHANGELOG

* Word missing on Readme.md (#592)

* make scss variables changeable

all variables should only provide a default

this would allow us to include the screen.scss and simply set the variables we want to change before that.

* Add Ruby 2.2.0 to Travis testing matrix

* fix -margin

* Add some company links to readme

* Remove unused gem middleman-gh-pages

* Update middleman-syntax

* Upgrade sprockets

* Fix another bug where disabling language tabs didn't properly hide HTML

* Update ruby version requirements in Travis and README

* Fix bug where -margin wasn't properly respected even if search was off

* Fix build, update middleman

* Exec middleman server fails with invalid flags --force-polling and -l. Removed the flags to remedy.

* Update language list link and count in README

* Add back in middleman flags to Vagrant with new flag syntax

* Fix incorrect documentation in deploy.sh

* Update pull requset template

* Cut version 1.4.0

* Add Ruby 2.3.3 to tested rubies

* Add multiple-tabs-per-language test

* Latest middleman - fixing startup arguments (#653)

As per https://github.com/middleman/middleman/issues/1866#issuecomment-221125503
Tested. It works.

* Update redcarpet gem to 3.4.0 which will solve the unicode error with (#660)

h1/h2.
See https://github.com/vmg/redcarpet/issues/538

* Updated Mozilla localForage link (#665)

Old link was broken

* Add Ruby 2.4.0 to .travis.yml

* Update middleman and middleman-sprokets, run bundle update

* Update code highlighting theme from Base16::Monokai to just Monokai

* Switch theme from Monokai to the less neon MonokaiSublime

* Allow Ruby 2.4.0 to fail for now

!!!!! STOP AND READ !!!!!

If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.

If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.
